lnutils/log.go

```diff
@@ -25,3 +29,10 @@ func SpewLogClosure(a any) LogClosure {
 		return spew.Sdump(a)
 	}
 }
+
+// NewSeparatorClosure return a new closure that logs a separator line.
+func NewSeparatorClosure() LogClosure {
+	return func() string {
+		return strings.Repeat("=", 80)
```

Comment:
yeah i like it, tho this is not as suitable for closure logs - this method will only be called when the debug log is called, and the it's on the stack so the mem is cleaned afterwards, whereas if we put the allocation outside, it will live on the heap.